olp-cpp-sdk-dataservice-read/src/QueryMetadataJob.h

```diff
@@ -27,25 +27,21 @@
 #include "ExtendedApiResponse.h"
 #include "TaskSink.h"
 
+#include <iterator>
```

Comment:
Obviously what applies to PrefetchPartitionsHelper.h also applies here concerning include order.